keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -77,6 +79,21 @@ File and block are first class citizens within the Kubernetes ecosystem.  Object
 
 # Proposal
 
+## User Stories
+
+#### Admin
+
+- As a cluster administrator, I can set generated buckets' storage capacity via the Kubernete's API, so that workloads may not inordinately contribute to monthly infrastructure costs.
```

Comment:
This is a good example but can we be more general? Eg.
`As a cluster administrator, I want to be able to control costs associated with object storage such that storage attributes can be limited, declined or otherwise managed.`